MEDMI_Data_Download/met_extraction_script.py

```diff
@@ -166,19 +411,25 @@ def create_directory(dir_name):
     parser = argparse.ArgumentParser(
         description="*** A script for automated downloading of MET data for a given date range. ***")
 
-    AVAILABLE_MEASUREMENTS = ['pollen', 'rain', 'temp',  'wind']
+    AVAILABLE_MEASUREMENTS = ['rain', 'temp', 'rel_hum', 'pressure', 'dewpoint', 'wind', 'pollen']
+    AVAILABLE_EXTRA_MEASUREMENTS = \
+        ['rain', 'temp', 'rel_hum', 'pressure', 'dewpoint', 'wind']# + MetExtractorPollen.pollen_species
+                                                                    # Pollen species input as extra data fail silently
+                                                                    # (the output just has no column for that data)
 
     ### parameters
 
-    # Measurement
+    # Measurements
     parser.add_argument("--measurements", "-m", metavar='M', type=str, nargs='+', help="measurements to be extracted. \
             Must be in (and defaults to) {}".format('[' + ", ".join([measure for measure in AVAILABLE_MEASUREMENTS]) + ']'))
-    #parser.add_argument("--measurements", "-m", dest="measurement", type=str, nargs='+',
-    #                    help="measurements to be extracted. Default: all")
+
+    parser.add_argument("--extra_measurements", "-x", metavar='X', type=str, nargs='+', help="extra measurements to be \
+        extracted along with each of the main measurements. Must be in {}. Default: None".format(
+        '[' + ", ".join([measure for measure in AVAILABLE_EXTRA_MEASUREMENTS]) + ']'))
```

Comment:
It doesn't make sense to me to apply the extra measurements to all the requested measurements. Can we change this so that you specify which measurement you want to add the extra measurements to? e.g.
--measurements temp+rain+pollen+wind
--extra_measurements temp:pressure+dewpoint+rel_hum

Reply:
To make it easier (for us at this point), we could just have an '--add_extra_datasets' flag that adds all extra datasets that are realistic for the requested measurement(s) and explain these in Readme. So
i) there is only one parameter, so less work required on parsing and matching measurements with extras
ii) measurements like pollens would not have any extras datasets
iii) Temp for example would have: humidity, dewpoint and pressure

Reply:
Okay - put can we restrict the extra datasets to just temp, pressure, dewpoint, and rel_hum? Both to as the extra datasets which are added, and the datasets selection to which extra datasets are added. (so wind and rain would be included in your point ii - measurements which have no extra datasets)